pkg/ocm/ocm.go

```diff
@@ -13,31 +13,51 @@ import (
 	hivev1 "github.com/openshift/hive/apis/hive/v1"
 )
 
-type ocmClient struct {
+//go:generate mockgen -source $GOFILE -destination mock/mockOcmHandler.go -package $GOPACKAGE ocmHandlerIf
+
+type ocmHandlerIf interface {
+	OcmGetResourceLive(clusterID, resourceKey string) (string, error)
+}
+type ocmHandler struct {
 	conn *sdk.Connection
-	cfg  *sdkcfg.Config
+}
+
+func (o ocmHandler) OcmGetResourceLive(clusterID, resourceKey string) (string, error) {
+	response, err := o.conn.ClustersMgmt().V1().Clusters().Cluster(clusterID).Resources().Live().Get().Send()
+	if err != nil {
+		return "", err
+	}
+	return response.Body().Resources()[resourceKey], nil
+}
+
+// OcmClient provides a client for the Openshift Cluster Manager
+type OcmClient struct {
+	//conn *sdk.Connection
+	Cfg  *sdkcfg.Config
```

Comment:
if it's public it should have a public docstring?

also the name seems to be too short for the scope this field has

Reply:
I think if we use a constructor for this one. We can get rid of the public fields as well

Reply:
yes, we don't need it at all actually afaik. We can get rid of it.

Reply:
I made the two fields private